fluency-fluentd/src/main/java/org/komamitsu/fluency/fluentd/ingester/sender/NetworkSender.java

```diff
@@ -124,9 +118,13 @@ protected synchronized void sendInternal(List<ByteBuffer> buffers, String ackTok
                 throw new SocketTimeoutException("Socket read timeout");
             }
 
-            Response response = objectMapper.readValue(optionBuffer, Response.class);
-            if (!ackToken.equals(response.getAck())) {
-                throw new UnmatchedAckException("Ack tokens don't matched: expected=" + ", got=" + response.getAck());
+            try (MessageUnpacker responseUnpacker = MessagePack.newDefaultUnpacker(optionBuffer)) {
+                responseUnpacker.unpackMapHeader();
+                responseUnpacker.unpackString();
```

Comment:
`ack` field is contained in a Map structure data, and the order and the number of fields can be changed https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1#response. So, the Ack value must be obtained by specifying key `ack`.

Reply:
Missed this one...  Fixed.